docs/portal/image-registry/managing-image-registry.md

```diff
@@ -0,0 +1,5 @@
+# Managing Registry Projects
```

Comment:
The contents in this page are inconsistent with previous pages. Please refer to this page "managing-image-pull-secrets" as an example. 

Note that the "Manaing Registry Users" page lacks the consistency as well.

Reply:
I fixed this.